server/routes/taggedBailRoutes.ts

```diff
@@ -176,10 +176,35 @@ export default class TaggedBailRoutes {
       })
     }
     const adjustment = this.adjustmentsStoreService.getById(req, nomsId, id)
+    const adjustments = await this.adjustmentsService.getAdjustmentsExceptOneBeingEdited(
+      { [id]: adjustment },
+      nomsId,
+      token,
+    )
     const sentencesAndOffences = await this.prisonerService.getSentencesAndOffences(bookingId, token)
 
+    const unusedDeductions = await this.calculateReleaseDatesService.unusedDeductionsHandlingCRDError(
+      { [id]: adjustment },
+      adjustments,
+      sentencesAndOffences,
+      nomsId,
+      token,
+    )
+
+    const showUnusedMessage = hasCalculatedUnusedDeductionDaysChangedFromUnusedDeductionAdjustmentDays(
```

Comment:
I think you should copy what the `remand.save` route does. An edit/delete/create will affect the unused deductions differently.

Reply:
I have followed the example from `remandRoutes.save`